source/includes/_ethereumfundamentalsv2.md

```diff
@@ -341,40 +341,40 @@ This endpoint returns the total and average fees spent on the Ethereum network f
 <img src="https://img.shields.io/badge/Tier-Hobbyist-blue.svg"/>
 
 ```shell
-curl "https://api.tokenanalyst.io/analytics/private/v1/token_miner_hashrate_window_historical/last?format=json&token=eth&key=API_KEY&window=1d"
+curl "https://api.tokenanalyst.io/analytics/private/v1/token_miner_hashrate_window_historical/last?format=json&token=eth&window=1d&from_date=2016-04-06&to_date=2016-04-07&limit=2&key=API_KEY"
 ```
 
 > The above command returns JSON structured like this:
 
 ```json
 [
   {
-    "date": "2016-04-07",
-    "miner": "0x5bD7a5f263fD5B00Fc8047D581fDBa92E34783b8",
-    "miner_name": "Unknown",
-    "total_daily_hashrate": 1.6977,
-    "miner_daily_hashrate": 0.0006,
-    "miner_daily_hashrate_pct": 0.0333,
-    "miner_daily_block_count": 2,
-    "miner_daily_uncle_count": 0,
-    "miner_daily_uncle_pct": 0,
-    "total_daily_block_count": 5953,
-    "total_daily_uncle_count": 403,
-    "total_daily_uncle_pct": 6
+    'date': "2016-04-07",
```

Comment:
why did you change from " to ' change

![image](https://user-images.githubusercontent.com/5904910/67559968-d6195580-f711-11e9-8caa-d5efb3e18d6c.png)

vs

![image](https://user-images.githubusercontent.com/5904910/67560003-e5000800-f711-11e9-9e23-299ef986f9fa.png)